posts/2024-07-30-24.0.0.8-beta.adoc

```diff
@@ -0,0 +1,808 @@
+
+[source,xml]
+----
+<webAppSecurity sameSiteCookie="None" partitionedCookie="defer|true|false"/>`
+----
+
+The following example shows how to set the `partitioned` attribute for other cookies the `httpEndpoint` attribute in your `server.xml` file:
+
+[source,xml]
+----
+<httpEndpoint id="defaultHttpEndpoint"
+              httpPort="9080"
+              httpsPort="9443" >
+   <samesite none="*" partitioned="true|false"/>
+</httpEndpoint>
+----
+
+
+Alternatively, you can set `Partitioned` by using the `Set-Cookie` header with the following two `HttpServletResponse` APIs:
+
+* link:https://openliberty.io/docs/ref/javaee/8/#package=javax/servlet/http/package-frame.html&class=javax/servlet/http/HttpServletResponse.html#setHeader-java.lang.String-java.lang.String-[HttpServletResponse.setHeader]
+* link:https://openliberty.io/docs/ref/javaee/8/#package=javax/servlet/http/package-frame.html&class=javax/servlet/http/HttpServletResponse.html#addHeader-java.lang.String-java.lang.String-[HttpServletResponse.addHeader]
+
+For more information, including a visual example, see link:https://github.com/privacycg/CHIPS?tab=readme-ov-file#chips-cookies-having-independent-partitioned-state[CHIPS (Cookies Having Independent Partitioned State)] on GitHub.
+
+[#ft]
+== Integrate fault tolerance with Open Telemetry in Fault Tolerance 4.1
+
+MicroProfile Fault Tolerance helps you easily identify and mitigate failures in your code. It provides annotations that you can add to methods to use bulkhead, circuit breaker, retry, timeout, and fallback strategies.
+
+The new `mpFaultTolerance-4.1` feature integrates with the `mpTelemetry-2.0` feature, so that Fault Tolerance can export metric data to Open Telemetry. With this change, and other changes in `mpTelemetry-2.0`, you can use Open Telemetry as the single source for logging, metrics, and tracing, making it easier to manage and configure your application observability.
+
+To enable this functionality, enable `mpFaultTolerance-4.1` and `mpTelemetry-2.0` in your `server.xml` file and then configure `mpTelemetry-2.0` to export metrics. The following examples show a minimal configuration for Open Telemetry to export to your `messages.log` file and a class that generates Fault Tolerance metrics (it must be accessed as a CDI bean).
+
+=== server.xml file configuration
+
+[source,xml]
+----
+<featureManager>
+  <feature>mpFaultTolerance-4.1</feature>
+  <feature>mpTelemetry-2.0</feature>
+</featureManager>
+----
+
+=== bootstrap.properties file configuration
+
+The following example configures Open Telemetry to only output metrics to the `messages.log` file.
+
+[source,xml]
+----
+otel.sdk.disabled=false
+otel.metrics.exporter=logging
+otel.logs.exporter=none
+otel.traces.exporter=none
+
+TODO; set the timeout to a low number so people will see the metrics quickly. Then mention why you did so in the paragraph apart

```

Comment:
Replace this TODO (and the empty line above it) with 

otel.metric.export.interval=500